### PR TITLE
Move wallet and add lock screen

### PR DESCRIPTION
**Pull request by tomsmalley**
The lock screen is currently totally unstyled, but it should work.

Also need to BEMify bits of sass.

### CODE REVIEW

File: frontend/src/Frontend/JsonData.hs [hunk 1]
```diff
@@ -306,11 +318,14 @@ makeKeysets walletL cfg =
...
         keys :: Dynamic t (Map KeyName PublicKey)
         keys = do
-          names <- Set.toList <$> keynames
-          let namesMap = Map.fromList . map (,()) $ names
-          wKeys <- walletL ^. wallet_keys
-          let keyPairs = Map.intersection wKeys namesMap
-          pure $ _keyPair_publicKey <$> keyPairs
+          names <- keynames
+          wAccounts <- walletL ^. wallet_accounts
+          -- This isn't great... should really come back to this
```
Comment (mankyKitty) [inline frontend/src/Frontend/JsonData.hs]: Fold over `IntMap.elems wAccounts` ? That would let you do the filtering and building of the new map in a single pass?

File: frontend/src/Frontend/UI/Wallet.hs [hunk 2]
```diff
@@ -122,117 +125,79 @@ uiAvailableKeys aWallet = do
...
   :: MonadWidget t m
-  => (Text, Dynamic t KeyPair)
-  -> m (Event t KeyName)
-uiKeyItem (n, k) = do
-    elClass "tr" "table__row" $ do
-      el "td" $ text n
-
-      let public = keyToText . _keyPair_publicKey <$> k
+  => IntMap.Key
+  -> Dynamic t (SomeAccount key) -- (Text, Dynamic t (KeyPair PrivateKey))
```
Comment (mankyKitty) [inline frontend/src/Frontend/UI/Wallet.hs]: leftover?

File: frontend/src/Frontend/UI/Wallet.hs [hunk 3]
```diff
@@ -122,117 +125,79 @@ uiAvailableKeys aWallet = do
...
 -- distinguish between the two at this point.
-getBalance :: (MonadWidget t m, HasNetwork model t) => model -> ChainId -> Event t AccountName -> m (Event t (Maybe Decimal))
+getBalance
+  :: (MonadWidget t m, HasNetwork model t, HasCrypto key (Performable m))
+  => model -> ChainId -> Event t AccountName -> m (Event t (Maybe AccountBalance))
 getBalance model chain account = do
   networkRequest <- performEvent $ attachWith mkReq (current $ getNetworkNameAndMeta model) account
   response <- performLocalReadCustom (model ^. network) pure networkRequest
-  let toBalance (_, [Right (_, Pact.PLiteral (Pact.LDecimal d))]) = Just d
+  let toBalance (_, [Right (_, Pact.PLiteral (Pact.LDecimal d))]) = Just $ AccountBalance d
```
Comment (mankyKitty) [inline frontend/src/Frontend/UI/Wallet.hs:200]: We should probably see if Pact has `Prism`s for this, that could be my preference leaking in though. 

```
toBalance = preview (_2 . _head . _Right . _2 . _PLiteral . _PDecimal. _Just . to AccountBalance)
```

Comment (tomsmalley) [reply frontend/src/Frontend/UI/Wallet.hs:200]: I don't think they do

File: frontend/src/Frontend/JsonData.hs (hunk 1 above)
Comment (tomsmalley) [reply frontend/src/Frontend/JsonData.hs]: I converted it to a fold, it's a little neater. I can't remember exactly what I was unhappy about.

**Pull request merged**